simplemma/lemmatizer.py

```diff
@@ -561,7 +297,7 @@ def lemmatize(
     if not silent:
         raise ValueError(f"Token not found: {token}")
     # try to simply lowercase # and len(token) < 10 ?
-    return token.lower() if lang[0] in BETTER_LOWER else token
+    return token.lower() if dictionaries[0].code in BETTER_LOWER else token
 
 
 def text_lemmatizer(
```

Comment:
This method uses the hardcoded `simple_tokenizer` in line 313.
What if the user needs to use a different tokenizer?
Either he has to override the `simple_tokenizer` method or implement a new `text_lemmatizer` using the raw `lemmatizer`

Reply:
I get your point but I didn't imagine people would actually rely on this function because it's so easy to use any existing tokenizer and then glue it to the lemmatization function. So I don't expect advanced users to work with this function but I'll think about it.

Feel free to open an issue since I understand you may be concerned (i.e. you prefer to use a customized function offered by Simplemma rather than an external one).